### DISCUSSION TITLE
Tile Animated GIFs

### DISCUSSION THREAD

**Discussion by kelsorj** [ImageMagick/ImageMagick#3449] [Digital Image Processing] [answered]
I don't know if this is possible with ImageMagick but here's the problem I'm trying to solve
animated1.gif
animated2.gif
....
animatedN.gif
Take all of those animated GIFs and put them side by side in a tiled grid NxN and have all of them continue to be animated.
I've tried:

montage -tile 1x2 animated1.gif animated2.gif output.gif

But that takes the two GIFs and in both tiles runs the first one and then runs the second one.
What am I missing to have tile 1 be the first GIF and tile 2 be the second GIF. I'm hoping this is possible using this tool or something else. Thanks for any help.
![out](https://user-images.githubusercontent.com/3299338/112648738-097c7a00-8e07-11eb-8186-42545e1ca4a2.gif)

**Comment by fmw42** [comment, marked as answer]
I do not believe you can use montage to do that.  See https://legacy.imagemagick.org/Usage/anim_mods/#merging.

**Reply by kelsorj** [reply]
Thanks so very much

convert point1.gif -repage 1024x512 -coalesce null: \( ooint1.gif -coalesce \) -geometry +512+0 -layers Composite out.gif

That got me what I needed. Now I've just got to script recursing through the directory to build up the tiled image.

https://user-images.githubusercontent.com/3299338/112659880-12bf1400-8e12-11eb-85bb-665d7610835c.mov
